draft-ietf-sedate-datetime-extended.md

```diff
@@ -377,14 +372,26 @@ number of these tags.
 Keys are lower-case only.
 Values are case-sensitive unless otherwise specified.
 
-When a suffix contains a repeated key or otherwise conflicting tags,
-implementations MUST give precedence to whichever value is positioned
-first. [^interop1]
+For compatibility with {{JAVAZDT}} and other existing implementations, a
```

Comment:
This section adds additional error-handling behavior, as recommended by @cabo as well as AD reviewer Francesca Palombini.